third_party/docfx-doclet-143274/src/main/java/com/microsoft/lookup/PackageLookup.java

```diff
@@ -110,6 +110,10 @@ public Optional<ApiVersion> extractApiVersion(String name) {
     return ApiVersion.parse(getLeafPackage(name));
   }
 
+  public boolean isApiVersionPackage(PackageElement pkg) {
+    return extractApiVersion(pkg).isPresent();
+  }

```

Comment:
nice. I will plan to refactor https://github.com/googleapis/java-docfx-doclet/pull/209 to use this instead (I did a flavor of this).